src/account_multisig.ts

```diff
@@ -118,11 +129,17 @@ export class AccountMultisig extends Account {
     }
 
     getRequest() {
-        return JSON.parse(localStorage.getItem(`__multisigRequest`) || `{}`)
```

Comment:
I think it's better to pass `storage` as optional parameter (e.g. as part of 3rd `options` parameter) when creating `AccountMultisig`. In browser user of this class can pass `localStorage` if needed.

Then you don't have to use same key in localstorage for every `AccountMultisig` instance in browser as any custom implementation can be passed.